samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -89,31 +91,38 @@ - (void)viewDidLoad {
     img = [UIImage imageNamed:[NSString stringWithFormat:@"photo%d.jpg", accelerateImageNumber]];
 }
 
+
 - (NSInteger)tableView:(UITableView*)tableView numberOfRowsInSection:(NSInteger)section {
     return 8;
 }
 
+
 - (CGFloat)tableView:(UITableView*)tableView heightForRowAtIndexPath:(NSIndexPath*)indexPath {
     CGRect screenRect = [[UIScreen mainScreen] bounds];
-    if (indexPath.row == 1)
+
+    if (indexPath.row == 1) {
         return screenRect.size.height - 360;
+    }
+
     return 40;
 }
 
+
 - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSIndexPath*)indexPath {
     UITableViewCell* cell = [tableView dequeueReusableCellWithIdentifier:@"MenuCell"];
+
     if (nil == cell) {
         cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"MenuCell"];
     }
     
+    // Title cell
     if (indexPath.row == 0) {
-        // Title cell
-        
-        cell.textLabel.text = @"Accelerate Test";
+        cell.textLabel.text = @"Accelerate Sample";
         cell.selectionStyle = UITableViewCellSelectionStyleNone;
-    } else if (indexPath.row == 1) {
-        // Image display box
-        
+    } 
+    
+    // Image display box
+    else if (indexPath.row == 1) {
```

Comment:
nit: our coding conventions put this in same line as { (not sure why clang format didn't do it automatically)